app/routes/people/person/personal-information.js

```diff
@@ -11,11 +17,99 @@ export default class PeoplePersonPersonalInformationRoute extends Route {
       include: [
         'mandatories.mandate.governing-body.is-time-specialization-of.administrative-unit.classification',
         'mandatories.contacts',
+        'agents-in-position',
       ].join(),
     });
+    const contacts = [];
+    const m = await person.mandatories;
+    console.log(m);
+    const mandatories = person.mandatories.filter(
+      (m) => m != null // hack because otherwise mandatories is not an array
+    );
+
+    const ministers = person.agentsInPosition.filter(
+      (m) => m != null // same
+    );
+
+    for (let mandatory of mandatories) {
```

Comment:
```suggestion
    for (let mandatory of mandatories.toArray()) {
```

Reply:
I did it in the assignement

Reply:
Hmm, normally it's not recommended to access async relationships without awaiting them first but as long as it works :+1:.